src/pages/update/Update.tsx

```diff
@@ -30,6 +35,11 @@ const Update = (props: Props) => {
 
   if (!incident) return <div>Loading...</div>
 
+  if (incident.updates.some(update => update.type === 'resolved')) {
+    // If it's already resolved, we go back to the incident status page
+    props.history.replace(`/${incident.id}`)
```

Comment:
FYI you could also use the `Redirect` component here. In cases where you don't have access to `history` for example

Reply:
You're right. Since I was using it inside `useEffect` I couldn't render it there, but now I can. Let me fix it.

Reply:
Done.